docs/ci/secrets.md

```
@@ -0,0 +1,373 @@
+```
+
+To verify an optional secret degrades:
+
+```bash
+# Clear SEMGREP_APP_TOKEN from GitHub Actions settings
+# Trigger CI workflow
+# Observe: ::warning message but build passes
+# Semgrep scans with reduced functionality (local rules only)
+```
+
+## Implementation Status
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)